lightning/src/routing/scoring.rs

```diff
@@ -386,19 +399,28 @@ pub struct ProbabilisticScoringParameters {
 	/// uncertainty bounds of the channel liquidity balance. Amounts above the upper bound will
 	/// result in a `u64::max_value` penalty, however.
 	///
+	/// `-log10(success_probability) * liquidity_penalty_multiplier_msat`
+	///
 	/// Default value: 40,000 msat
 	///
 	/// [`liquidity_offset_half_life`]: Self::liquidity_offset_half_life
 	pub liquidity_penalty_multiplier_msat: u64,
 
-	/// The time required to elapse before any knowledge learned about channel liquidity balances is
-	/// cut in half.
+	/// Whenever this amount of time elapses since the last update to a channel's liquidity bounds,
+	/// the distance from the bounds to "zero" is cut in half. In other words, the lower-bound on
+	/// the available liquidity is halved and the upper-bound moves half-way to the channel's total
+	/// capacity.
+	///
+	/// Because halving the liquidity bounds grows the uncertainty on the channel's liquidity,
+	/// penalties for payments that are within the liquidity bounds will be decreased. See the
+	/// [`ProbabilisticScorer`] struct documentation for more info on the way the liquidity bounds
+	/// are used.
 	///
-	/// The bounds are defined in terms of offsets and are initially zero. Increasing the offsets
-	/// gives tighter bounds on the channel liquidity balance. Thus, halving the offsets decreases
-	/// the certainty of the channel liquidity balance.
+	/// For example, if the channel's capacity is 1 million sats, and the current upper- and lower-
+	/// liquidity bounds are 200,000 sats and 600,000 sats, after this amount of time the upper-
```

Comment:
nit: Move "after this amount of time" to the end of the sentence. Also, since this is an example, it would be nice to make it more concrete, e.g., use the default values or variable name?

Reply:
Alright, still think would read better if "after this amount of time"  would be at the end of the sentence, but feel to leave as is if you prefer.